<launch>

  <param name="robot_description" textfile="$(find rpbi_examples)/robots/kuka_lwr.urdf"/>

  <node pkg="ros_pybullet_interface" name="ros_pybullet_interface" type="ros_pybullet_interface_node.py">
    <rosparam param="config" file="$(find optas_ros)/example/example_controller_config.yaml"/>
  </node>

  <node pkg="optas_ros" name="controller" type="optas_controller_node.py"/>
  <node pkg="optas_ros" name="master" type="example_controller_node.py"/>
  
</launch>
